src/main/webapp/availability.js

```diff
@@ -66,3 +72,23 @@ function updateAvailability() {
   let request = new Request('/availability', {method: 'PUT', body: requestBody});
   fetch(request).then(unused => {loadAvailabilityTable(availabilityTableDiv(), browserTimezoneOffset())});
 }
+
+let page = 0;
+
+function goBack() {
+  if (page <= 0) {
+    page = 0;
+    return;
+  }
+  page -= 1;
+  loadAvailabilityTable(availabilityTableDiv(), browserTimezoneOffset());
+}
+
+function goForward() {
+  if (page >= 3) {
```

Comment:
Can you move this `3` to a constant somewhere?

Reply:
Done